test/e2e/antreapolicy_test.go

```diff
@@ -1944,7 +1944,10 @@ func testANPMultipleAppliedTo(t *testing.T, singleRule bool) {
 	}
 
 	t.Logf("Making the Policy apply to y/c by labeling it with the temporary label that matches the dummy AppliedTo")
-	podYC, _ := k8sUtils.GetPodByLabel("y", "c")
+	podYC, err := k8sUtils.GetPodByLabel("y", "c")
+	if err != nil {
+		t.Errorf("no pod matches in ns y with label 'pod=c'")
```

Comment:
```suggestion
		t.Errorf("Failed to get Pod in Namespace y with label 'pod=c': %v", err)
```
Because the error could be caused by connection issue too. It should use the returned error as the reason.